commands/pipelines/info.js

```diff
@@ -3,8 +3,18 @@
 const co = require('co')
 const cli = require('heroku-cli-util')
 const disambiguate = require('../../lib/disambiguate')
-const stageNames = require('../../lib/stages').names
 const listPipelineApps = require('../../lib/api').listPipelineApps
+const getTeam = require('../../lib/api').getTeam
+const sortBy = require('lodash.sortby')
+
+// For user pipelines we need to use their couplings to determine user email
+function getUserPipelineOwner (apps, userId) {
+  for (let app in apps) {
```

Comment:
It's not likely, but possible that the none of the apps the current user is collaborated on are owned by the pipeline owner. In other words, there may not be an app in the list with `userId` as the owner here. I'd suggest returning the `userId` here if we can't resolve an email.

Reply:
I think what you mean is that if pipeline owner doesn't own any application in the pipeline, but I agree, we should at least return the uuid there.

Reply:
@joshwlewis https://github.com/heroku/heroku-pipelines/pull/67/commits/09ccd4070516eee9b76f3918b43548ca4abca8c8